ts/.eslintrc.base.json

```diff
@@ -16,6 +17,17 @@
       "tsconfig.json"
     ]
   },
+  "settings": {
+    "import/parsers": {
```

Comment:
By default `eslint-plugin-import` can't detect correctly `exports` section to detect esm, this seems to resolve the issue